extensions/grpc/runtime/src/main/java/io/quarkus/grpc/runtime/supports/Channels.java

```diff
@@ -208,6 +209,19 @@ public static Channel createChannel(String name, Set<String> perClientIntercepto
 
             if (builder instanceof NettyChannelBuilder) {
                 NettyChannelBuilder ncBuilder = (NettyChannelBuilder) builder;
+                // just use the existing Vertx event loop group, if possible
+                Vertx vertx = container.instance(Vertx.class).get();
+                // only support NIO for now, since Vertx::transport is not exposed in the API
+                if (vertx != null && vertx.isNativeTransportEnabled()) {
```

Comment:
@vietj @cescoffier if we can query `Vertx` re which transport is using we can enable native transport Epoll as well since is the one supported by grpc java (with Netty).

Reply:
We could imagine checking the configuration and whether a native transport was found in the classpath at build time.

Reply:
I still consider this option (i.e. not using Vertx GrpC client) although the default choice, not what we would like users to go with, in the future.
So I can leave this additional config out for now :pray: